ci-operator/config/openshift/installer/openshift-installer-release-4.12.yaml

```
@@ -502,13 +502,20 @@ tests:
   steps:
     cluster_profile: ibmcloud
     workflow: openshift-e2e-ibmcloud
-- as: e2e-sdn-nutanix
+- as: e2e-nutanix
```

I think this should be
```suggestion
- as: e2e-nutanix-ovn
```
Right @jluhrsen ?

---

@patrickdillon Done

---

right, that is the pattern that was decided at some point.  e2e-<platform>-<cni>